scripts/deploy.sh

```diff
@@ -14,7 +15,7 @@ OSAD_DIR='/opt/rpc-extras/os-ansible-deployment'
 RPCD_DIR='/opt/rpc-extras/rpcd'
 
 # begin the bootstrap process
-cd "${OSAD_DIR}"
+cd ${OSAD_DIR}
```

Comment:
they were removed as they add no real value, and it felt more consistent to me to have them used in this way (this is how it's done in the OSAD scripts)